src/web.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="es">
+  <head>
+    <meta charset="UTF-8">
+    <META http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>API</title>
+    </head>
+        <body>
+          <h1>Lista de Eventos</h1>
+          <div class="container">
+
+          </div>
+          <script src="web.js"></script>
+
+        </body>
+</html>
```

Comment:
Se puede poner mas ordenado si lo ves bien:

```suggestion
<!doctype html>
<html lang="es">
<head>
    <meta charset="UTF-8">
    <META http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>API</title>
</head>
<body>
    <h1>Lista de Eventos</h1>
    <div class="container" />
    <script src="web.js"></script>
</body>
</html>
         
```

Reply:
Ctrl + K + D y Visual Studio te lo ordena